src/cmd/flux-jobs.py

```diff
@@ -42,7 +42,9 @@ def runtime(job, roundup):
 
 def runtime_fsd(job, hyphenifzero):
     t = runtime(job, False)
-    if t < 60.0:
+    if t >= 1e-3 and t < 10.0:
```

Comment:
I think there's a logic hole for `< 1e-3`.  Do we just want to round down and call it `0s`?

Reply:
oh wait, nevermind, it'll fall through to `else:` and `0s`.

Reply:
no, i'm getting my functions mixed up, the fallthrough would be the `days` else case.